app/src/main/AndroidManifest.xml

```diff
@@ -16,26 +15,12 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.DanXiNative"
         android:usesCleartextTraffic="true"
-        tools:targetApi="s">
-        <activity
-            android:label="关于"
-            android:name=".AboutActivity"
-            android:exported="false"/>
-        <activity
-            android:name=".SingleFragmentActivity"
-            android:exported="false" /> <!-- Opt-out the usage statistics collection by Google. -->
```

Comment:
Keep this comment about `android.webkit.WebView.MetricsOptOut`.

Reply:
finished, may some bug of GitHub